tests/foreman/api/test_product.py

```diff
@@ -378,12 +378,11 @@ def test_positive_filter_product_list(self):
             query={'redhat_only': True}
         )
 
-        self.assertEqual(len(custom_products), 1)
-        self.assertEqual(product.name, custom_products[0].name)
-        self.assertNotIn('Red Hat Beta', [prod.name for prod in custom_products])
-        self.assertGreater(len(rh_products), 1)
-        self.assertNotIn(product.name, [prod.name for prod in rh_products])
```

Comment:
Why this assertion isn't ported?

Reply:
@mirekdlugosz what do you mean?

Reply:
@latran  `self.assertNotIn(product.name, [prod.name for prod in rh_products]) ` is missing in current test. we just need to add `assert product.name not in (prod.name for prod in rh_products)` assertion and we are good.

Reply:
done.